src/components/organisms/Header.tsx

```diff
@@ -55,20 +55,21 @@ export const Header: React.FC = () => {
 
   return (
     <header
-      className={`grid grid-cols-[minmax(165px,_1fr)_minmax(min-content,696px)_minmax(165px,_1fr)] border-b border-solid border-gray-dark bg-background/80 px-4 pb-[calc(0.75rem_-_1px)] pt-3 backdrop-blur-[10px] sm:px-6`}
+      className={`grid grid-cols-[minmax(165px,_1fr)_minmax(min-content,696px)_minmax(165px,_1fr)] border-b border-solid border-gray-dark bg-background/10 px-4 py-4 backdrop-blur-[10px] sm:flex sm:gap-4 sm:px-6`}
     >
       <h1 className={`col-start-1 justify-self-start ${hideWhenSearch}`}>
         <Link to="/">
-          <div className="aspect-[10/6.97] h-10 sm:hidden">
+          <div className="hidden aspect-[10/6.97] h-10 sm:block">
             <VeridaOneLogo height="100%" width="100%" />
           </div>
-          <div className="hidden aspect-[10/3] h-10 sm:block">
+          <div className="aspect-[10/3] h-10 sm:hidden">
```

Comment:
Not quite sure of the justification of these changes on the header. Yes, the one on https://verida.one is different than the one on https://demo.verida.one but I specified in the ticket to not change it.

That being said we can fine a middle ground for the opacity. Either be a value working on every page, applying `10` on the Home page and `80` on the others.

Anyway, the layout is broken. The the narrow layout is applied to wide window and the wide layout is applied to the narrow window. The Search is particularly impacted.

The positioning with the SearchResult is also impacted

Reply:
I reverted the changes